src/main/java/org/sagebionetworks/bridge/models/schedules/ScheduleContext.java

```diff
@@ -85,6 +85,10 @@ public DateTime getStartsOn() {
         return startsOn;
     }
     
+    public DateTimeZone getRequestTimeZone() { 
+        return startsOn.getZone();
+    }
+    
```

Comment:
Do we also want to ensure startsOn and endsOn are in the same time zone?

Reply:
This is checked at ScheduledActivityController line 144. And it's a bug insofar as when time zones change, these can be different and then the query fails (BRIDGE-2126). Client developers have to force them to be the same.